content/en/aws/docdb/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "DocumentDB (DocDB)"
+linkTitle: "DocumentDB (DocDB)"
+categories: ["LocalStack Pro"]
+description: >
+  DocumentDB (DocDB)
+---
+
+LocalStack supports a basic version of [Amazon DocumentDB](https://aws.amazon.com/documentdb/) for testing.
```

Comment:
Can you please add a line or two about DocumentDB just for the sake of folks who are starting with this for the first time.